tracks/go/exercises/scrabble-score/README.md

```diff
@@ -0,0 +1,87 @@
+# Scrabble Score
+
+This is the first more complex task. There are several ways to solve this and simple and readable code are an important criteria on this solution. Benchmarks could also be introduced as a measurement of which solution is better.
```

Comment:
I'm concerned about including where an exercise falls in the core sequence in the mentoring guides, although I understand why it could be valuable. What do you mean by "more complex task"?

"Benchmarks could also be introduced as a measurement of which solution is better." <= Could you please explain?

Reply:
"more complex task" => happy to take this out. I guess it is a left over of previous versions. What I meant: In the core track this is the first task where the student goes beyond if statements (although he does not have to). But I have trouble explaining it... so lets scratch that.

"Benchmarks could also be introduced as a measurement of which solution is better."
When guiding the student to a simpler/better solution I find it helpful to have a measure. Why is the one solution he provided better his other solution? One criteria can be speed.

Reply:
Ah, gotcha.

Ideally, we would always have criteria (external measures/references) for the feedback we give. Benchmarks are certainly one we should use; for all exercises, not just for this one 😄

Reply:
"more complex task" <= Here's a thought. This exercise is the first in which we commonly see students write two functions.

Reply:
"This exercise is the first in which we commonly see students write two functions."
I like it. Will add that we see them using a `map`.